src/main/resources/static/api-docs.json

```diff
@@ -15,10 +15,389 @@
+          }
+        },
+        "security": [
+          {
+            "oauth2": []
+          },
+          {
+            "basicAuth": []
+          }
+        ]
+      }
+    },
+    "/v1/import/{importId}/{stagedElementId}": {
+      "get": {
+        "tags": [
+          "Import and StagedElement Management"
+        ],
+        "summary": "Get StagedElement from a specified Import by ID.",
+        "operationId": "Get StagedElement from a specified Import by ID.",
+        "parameters": [
+          {
+            "name": "importId",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "stagedElementId",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default response",
+            "content": {
+              "*/*": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "oauth2": []
+          },
+          {
+            "basicAuth": []
+          }
+        ]
+      }
+    },
+    "/v1/import/{importId}/{stagedElementId}/members?onlyConverted={onlyConverted}": {
+      "get": {
+        "tags": [
+          "Import and StagedElement Management"
+        ],
+        "summary": "Get StagedElement Members from a specified Import by ID.",
+        "operationId": "Get StagedElement Members from a specified Import by ID.",
+        "parameters": [
+          {
+            "name": "importId",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "stagedElementId",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "onlyConverted",
+            "in": "path",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "default": false
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default response",
+            "content": {
+              "*/*": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "oauth2": []
+          },
+          {
+            "basicAuth": []
+          }
+        ]
+      }
+    },
+    "/v1/import/{importId}/convert": {
+      "post": {
+        "tags": [
+          "Import and StagedElement Management"
+        ],
+        "summary": "Get StagedElement from a specified Import to Draft.",
```

Comment:
I didn't mark all in this file...but please check for correct capitalization

Reply:
Done